distribution/docs/src/main/resources/content/_running/starting-intro.adoc

```diff
@@ -238,21 +238,26 @@ From the https://lucene.apache.org/solr/guide/7_4/taking-solr-to-production.html
 Solr includes a service installation script (bin/install_solr_service.sh) to help you install Solr as a service on Linux. Currently, the script only supports CentOS, Debian, Red Hat, SUSE and Ubuntu Linux distributions.
 ====
 
-. Extract the Installation Script
+. Locate the Installation Script
 +
-Extract the Solr distribution installation script (install_solr_service.sh) into the current directory. By default they are located in  the `${home_directory}/distribution/solr/target/classes/solr-${solr.version}/bin` directory.
+Identify the Solr distribution installation script (install_solr_service.sh) in the solr directory. By default they are located in  the `${home_directory}/solr/bin` directory.
++
+
+. Prepare the Solr distribution
++
+Move all the contents of the solr directory into a new folder named solr-_version_ (where _version_ is `7.4.0` or whichever release is being used) and compress the folder into a tgz or zip.
 +
 
 . Run the installation script as root:
 +
 -----
-sudo bash ./install_solr_service.sh solr-version.tgz
+sudo bash ./install_solr_service.sh solr-version.tgz -p 8994
```

Comment:
❓ Do we need consider documenting why port `8994`? Can another port be used? If so what are the impacts?

Reply:
That is the just the port we have selected to run Solr on. Another port can be specified in this step and in the previous version I didn't have any specific port listed. I was hoping that the user would be inclined to make their own decision about what port to use and understand that it needed to match the configuration in DDF in the `system.properties` file. I received feedback from several people running through these steps that they used the suggested commands without specifying a port or updating the properties. This meant that Solr was starting on a different port from where DDF was looking for it. 

I updated the command to include the default port in hopes than anyone copying pasting directly from docs without updating their properties will be more likely to have a successful installation.

I will add another note below this step explaining that the system properties should match the port they select to start Solr.